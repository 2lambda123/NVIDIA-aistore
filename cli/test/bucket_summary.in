ais show bucket
ais show bucket tmp // FAIL
ais create bucket $BUCKET_1
ais create bucket $BUCKET_2
ais show bucket
ais show bucket $BUCKET_1

ais gen-shards --bucket $BUCKET_1 --ext .tar --template="tmp/shard-{0..149}" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais gen-shards --bucket $BUCKET_2 --ext .tar --template="tmp/shard-{0..19}" --fcount 10 --fsize 1KB --conc 1 --cleanup // IGNORE
ais show bucket
