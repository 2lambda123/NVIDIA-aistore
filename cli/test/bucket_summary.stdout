Name	Objects	Size	Used(%)	Provider
"$BUCKET_1" bucket created
"$BUCKET_2" bucket created
Name	Objects	Size	Used(%)	Provider
$BUCKET_1	0	0B	0.00%	ais
$BUCKET_2	0	0B	0.00%	ais
Name	Objects	Size	Used(%)	Provider
$BUCKET_1	0	0B	0.00%	ais
Name	Objects	Size	Used(%)	Provider
$BUCKET_1	150 375.00KiB	0.00%	ais
$BUCKET_2	20 320.00KiB	0.00%	ais
